README.md

```diff
@@ -75,18 +75,26 @@ COPY --chown=1001:0 <path_to_customized_snippet> /config/configDropins/overrides
 
 It is important to be able to observe the logs emitted by Open Liberty when it is running in docker. A best practice method would be to emit the logs in JSON and to then consume it with a logging stack of your choice.
 
-Configure your Open Liberty docker image to emit JSON formatted logs to the console/standard-out with your selection of liberty logging events by providing the following environment variables to your Open Liberty DockerFile.
-
-For example:
+Configure your Open Liberty docker image to emit JSON formatted logs to the console/standard-out with your selection of liberty logging events by creating and adding a `bootstrap.properties` file to your Open Liberty Dockerfile.
+```dockerfile
+COPY --chown=1001:0  bootstrap.properties /config/
```

Comment:
this dockerfile text should probably go after showing the bootstrap.properties file we need them to create.